01_RecordSelectors/popup/popup.js

```
@@ -98,6 +127,80 @@ document.addEventListener('DOMContentLoaded', () => {
+                console.error("[Popup] Error sending stop screen recording message:", chrome.runtime.lastError.message);
+             }
+        });
+    });
+
+     downloadVideoBtn.addEventListener('click', () => {
+        chrome.runtime.sendMessage({ action: 'getRecordedVideoUrl' }, response => {
+            if (chrome.runtime.lastError) {
+                console.error("Error getting video URL:", chrome.runtime.lastError.message);
+                return;
+            }
+            if (response && response.url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)